src/engine/QueryPlanner.cpp

```diff
@@ -723,7 +724,8 @@ vector<QueryPlanner::SubtreePlan> QueryPlanner::seedWithScansAndText(
     using enum Permutation::Enum;
 
     if (node._cvar.has_value()) {
-      seeds.push_back(getTextLeafPlan(node));
```

Comment:
Can you add the trivial function to the `node` class, s.t. we can write
`if (node.isTextNode())` here?